apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/KafkaProducerProperty.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore;
+
+import com.expediagroup.apiary.extensions.events.metastore.common.Property;
+import com.expediagroup.apiary.extensions.events.metastore.common.io.jackson.JsonMetaStoreEventSerDe;
+
+public enum KafkaProducerProperty implements Property {
+  TOPIC("topic", null),
+  BOOTSTRAP_SERVERS("bootstrap.servers", null),
+  CLIENT_ID("client.id", "CircusTrainEventDrivenEmitter"),
```

Comment:
Not sure if we actually use this anywhere so either remove or change to something like "ApiaryKafkaMetaStoreListener" (of the fully qualified class name of it).